runtime/doc/vim_diff.txt

```diff
@@ -390,6 +390,8 @@ Highlight groups:
   using |n| or |N|
   |hl-CursorLine| is low-priority unless foreground color is set
   |hl-VertSplit| superseded by |hl-WinSeparator|
+  It is an error to define a highlight group with a name that doesn't match
+  the regexp `[a-zA-Z0-9_.@]*` (see |group-name|).
```

Comment:
Should we also mention that we allow "." and "@"?

Reply:
You mean, state explicitly how our regex is more liberal than Vim's? Yes, that's a good idea.